E-Commerce/readme.md

```diff
@@ -46,13 +46,27 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+           =>SELECT * FROM products WHERE product_name LIKE '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+          =>SELECT p.id AS product_id, p.product_name AS name, pa.unit_price, pa.supp_id AS supplier_id FROM products AS p INNER JOIN product_availability AS pa ON p.id = pa.prod_id WHERE pa.unit_price>100;
 - [ ] List the 5 most expensive products
+        =>select product_name, unit_price from products join product_availability on products.id = product_availability.prod_id order by unit_price desc limit 5;
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
+      => select product_name, supplier_name from products join product_availability on products.id = product_availability.prod_id join suppliers ON supp_id = suppliers.id where country = 'United Kingdom';  
 - [ ] List all orders, including order items, from customer named Hope Crosby
+        select * from  orders join order_items on orders.id = order_items.order_id join customers on orders.customer_id = customers.id WHERE customers.name = 'Hope Crosby';
+
+
 - [ ] List all the products in the order ORD006. The result should only contain the columns product_name, unit_price, and quantity
+            => select product_name, unit_price, quantity from orders join order_items on orders.id = order_items.order_id join products on order_items.product_id = products.id join product_availability on order_items.product_id = product_availability.prod_id and order_items.supplier_id = product_availability.supp_id where orders.order_reference = 'ORD006';
```

Comment:
really nice job with this one! I know it was a bit tricky, but you were able to get the joins and order_reference nicely.

Reply:
This was tricky, but nice catch with the two foreign keys: `order_items.product_id = product_availability.prod_id and order_items.supplier_id = product_availability.supp_id `